keyboards/kbdfans/kbdpad/mk2/mk2.h

```diff
@@ -27,17 +27,17 @@
  */
 #define LAYOUT_ortho_6x4( \
```

Comment:
Also supports `numpad_6x4`

Reply:
@ai03-2725 could you help with this one?
Which 3 pads are not used with the numpad_6x4 layout on this pcb?

Reply:
<img width="442" alt="image" src="https://user-images.githubusercontent.com/43021794/148164759-3e109305-2e84-43d5-a13c-738b311ec41a.png">